docs/users/cli-usage.md

```diff
@@ -25,13 +25,13 @@ ddev config
 
 _Note: ddev config will prompt you for a project name, docroot, and app type._
 
-After you've run `ddev config`, you're ready to start running your project. To start running ddev, simply enter:
+After you've run `ddev config`, you're ready to start running your project. To start running ddev, enter:
 
 ```
 ddev start
 ```
 
-When running `ddev start` you should see output informing you that the project's environment is being started. If startup is successful, you'll see a message like the one below telling you where the project can be reached.
+When `ddev start` runs, it outputs status messages to indicate the project environment is starting. When the startup is complete, ddev outputs a message like the one below with a link to access your project in a browser.
```

Comment:
Aside: the example output is out of date because we now have http/https.
```
Successfully started wordpress
Your project can be reached at http://wordpress.ddev.local and https://wordpress.ddev.local
```

Reply:
That works!